### PR TITLE
Remove sport_type_id, only use sport_type directly

### PR DESCRIPTION
**Pull request by TheWudu**
Get rid of sport_type_id, it is not necessary imo.

### CODE REVIEW

File: lib/sport_type.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 class SportType
   def self.id_for(name: nil)
+    raise 'do not use this'
     return 5 unless name
 
     SPORT_TYPE_MAPPING[name]
```
Comment (ghsmaniotto) [inline lib/sport_type.rb]: Doing like this, we are saying that ` swimming` and `open_water_swimming` are different sport types, right? Should we keep them as a same sport type?

Comment (TheWudu) [reply lib/sport_type.rb]: I think we can still map this, but I would not use an id mapping for this.

SportType.unified(name: name) 

Maybe? Returning swimming for swimming and oprn water swimming, and maybe other mappings too.

Comment (TheWudu) [reply lib/sport_type.rb]: Updated the PR

File: lib/sport_type.rb [hunk 2]
```diff
@@ -1,44 +1,45 @@
...
+  def self.unified(name:)
+    SPORT_TYPE_UNIFIED[name] || name
   end
 
-  def self.name_for(id: nil)
+  def self.name_for_runtastic_id(id: nil)
     return 'other' unless id
 
-    SPORT_TYPE_MAPPING.key(id)
+    SPORT_TYPE_MAPPING.fetch(id)
```
Comment (ghsmaniotto) [inline lib/sport_type.rb]: `RUNTASTIC_SPORT_TYPE_MAPPING.fetch(id)` instead of `SPORT_TYPE_MAPPING`

File: lib/sport_type.rb (hunk 1 above)
Comment (ghsmaniotto) [reply lib/sport_type.rb]: I liked the way it is now!

**Review by ghsmaniotto — APPROVED** [review]

File: lib/sport_type.rb (hunk 2 above)
Comment (TheWudu) [reply lib/sport_type.rb]: Fixed + added tests for runtastic-json parser

**Pull request merged**